nameko_grpc/headers.py

```diff
@@ -83,6 +84,24 @@ def comma_join(values):
     return separator.join(values)
 
 
+_UNQUOTED = "".join(
+    [chr(i) for i in range(0x20, 0x24 + 1)] + [chr(i) for i in range(0x26, 0x7E + 1)]
+)
+
+
+def encode_grpc_message(message):
```

Comment:
The names of this functions could be improved, I think.

"gRPC message" sounds like it refers to the `Message → *{binary octet}` part of the DATA frame. Actually what we're doing here is making a value safe to be included in gRPC headers.

Perhaps simply `quote_encode(value)` and `quote_decode(value)` are better signatures.